ui/src/app/pages/login/account-creation/account-creation-institution.tsx

```diff
@@ -0,0 +1,330 @@
+          <div style={{...styles.text, fontSize: 16, marginTop: '1rem'}}>
+            Please complete Step 1 of 3
+          </div>
+          <div style={{...styles.text, fontSize: 14, marginTop: '0.7rem'}}>
+            For access to the <i>All of Us</i> Research Program data, your institution needs to have signed a Data Use Agreement
+            with the program. The institutions listed below have an Institutional Data Use Agreement with the program that
+            enables us to provide their researchers with access to the Workbench.
+          </div>
+          <div style={{...styles.text, fontSize: 12, marginTop: '0.5rem'}}>
+            All fields are required.
+          </div>
+          {loadingInstitutions && <SpinnerOverlay />}
+          {!loadingInstitutions && <div style={{marginTop: '.5rem'}}>
+            <label style={{...styles.text, fontWeight: 600}}>
+              Select your institution
+              <i style={{...styles.publiclyDisplayedText, marginLeft: '0.2rem'}}>
+                Publicly displayed
+              </i>
+            </label>
+            <div style={{...styles.text, fontSize: 14}}>
+              Your institution will be notified that you have registered using your institutional credentials.
+            </div>
+            <Dropdown
+                data-test-id='institution-dropdown'
+                style={{width: '50%', minWidth: '600px'}}
+                options={institutions.map(inst => ({'value': inst.shortName, 'label': inst.displayName}))}
+                value={institutionShortName}
+                onChange={(e) => {
+                  this.updateAffiliationValue('institutionShortName', e.value);
+                  // Clear out any existing values for role when the institution changes.
+                  this.updateAffiliationValue('institutionalRoleEnum', undefined);
+                  this.updateAffiliationValue('institutionalRoleOtherText', undefined);
+                }}/>
+            {this.state.dataLoadError &&
+            <ErrorDiv data-test-id='data-load-error'>
+              An error occurred loading the institution list. Please try again or contact
+              <a href='mailto:support@researchallofus.org'>support@researchallofus.org</a>.
+            </ErrorDiv>
+            }
+            <div style={{marginTop: '.5rem'}}>
+              <a href={'https://www.researchallofus.org/apply/'} target='_blank' style={{color: colors.accent}}>
+              Don't see your institution listed?
+              </a>
+            </div>
+            <TextInputWithLabel containerStyle={{marginTop: '1rem', width: null}}
+                                value={contactEmail}
+                                inputId='contact-email'
+                                inputName='contactEmail'
+                                labelContent={<div>
+                                  <label style={{...styles.text, fontWeight: 600}}>
```

Comment:
Nit: There's a number of 'text but with fontWeight 600'. Would it make sense to create a `boldText` style, and use it everywhere that happens?

Reply:
done